sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -563,3 +592,52 @@ def import_key(self, name, key, **kwargs):
             **kwargs
         )
         return KeyVaultKey._from_key_bundle(bundle)
+
+    @distributed_trace
+    def export_key(self, name, version, parameters, **kwargs):
+        # type: (str, str, KeyExportParameters, **Any) -> KeyVaultKey
+        """Exports a key.
+
+        The export key operation is applicable to all key types. The target key must be marked
+        exportable. This operation requires the keys/export permission.
+
+        :param str name: The name of the key to export.
+        :param str version: A specific version of the key to export.
+        :param parameters: The parameters for the export operation.
```

Comment:
We might want to include the name and version in the parameters, rather than all three. In fact, in .NET we'd have to change the order anyway, since `version` is optional and we can't have required parameters after optional parameters. But looking at similar cases, we tend to just consolidate into an "options" or "parameters" (in this case, since it's required, probably the latter) parameter.

Reply:
The remaining parameters are all optional according to the latest [swagger changes](https://github.com/Azure/azure-rest-api-specs/pull/13913) so I think it makes the most sense to make them kwargs here and drop the class. For other languages that use a parameter object pattern, it might make more sense to move the key name/version into `KeyExportParameters`, like you said.

Reply:
We actually hoist up some optional parameters like `version` because they are important and common. We do that in other APIs. Maybe not `parameters`, but it's something to discuss further in the channel. For now, not blocking.